twilight-lavalink/src/node.rs

```diff
@@ -489,14 +521,24 @@ impl Connection {
         let (to_node, from_lavalink) = mpsc::unbounded_channel();
         let (to_lavalink, from_node) = mpsc::unbounded_channel();
 
+        let mut client_builder = HyperClient::builder(TokioExecutor::new());
+
+        if config.enable_tls {
+            client_builder.http2_only(config.enable_tls);
+        }
```

Comment:
You're not wrapping the connector inside a `hyper-rustls` or `hyper-tls` connector, so this won't enable TLS.